fast64_internal/oot/oot_upgrade.py

```diff
@@ -1,5 +1,6 @@
 from dataclasses import dataclass
 from typing import TYPE_CHECKING
+import bpy
```

Comment:
is this intended? looks like it's the only change in this file

Reply:
Yes, this is a bugfix, `bpy` was already used in this file (not in my code, not related to the rest of this PR) but it was an error when that was executed.